src/app/core/config.py

```diff
@@ -0,0 +1,13 @@
+import os
+from pydantic import AnyHttpUrl, BaseSettings
+
+class Settings(BaseSettings):
+    API_V1_STR: str = "/api/v1"
+    SERVER_NAME: str = os.getenv("SERVER_NAME", "sqe_backend")
+    SERVER_HOST: AnyHttpUrl = os.getenv("SERVER_HOST", "http://localhost")
+    PROJECT_NAME: str = os.getenv("PROJECT_NAME", "SQE-Welding-Project")
+    # ToDo: store and access secrets properly
+    DATABASE_URL_ASYNC: str = os.getenv("DATABASE_URL_ASYNC", f"postgresql+asyncpg://postgres:awesomepw@localhost:5432")
+    DATABASE_URL_SYNC: str = os.getenv("DATABASE_URL_SYNC", "postgresql://postgres:awesomepw@localhost:5432")
```

Comment:
We should have two newlines around this top level class definition according to [PEP8](https://peps.python.org/pep-0008/#blank-lines).

Reply:
Fixed